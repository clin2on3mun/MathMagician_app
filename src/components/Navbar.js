import React from 'react';
import { Link } from 'react-router-dom';

const Navbar = () => (
  <header className="header">
    <h1 className="header-title">Math Magician App</h1>
    <nav>
      <ul>
        <li><Link to="/">Home</Link></li>
        <li><Link to="/calculator">Calculator</Link></li>
        <li><Link to="/quotes">Quote</Link></li>
      </ul>
    </nav>
  </header>
);

export default Navbar;
