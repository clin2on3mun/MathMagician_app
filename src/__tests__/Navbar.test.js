import { render } from '@testing-library/react';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

describe('Navbar component', () => {
  test('renders correctly', () => {
    const { container } = render(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
