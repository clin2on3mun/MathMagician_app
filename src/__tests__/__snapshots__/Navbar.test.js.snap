// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component renders correctly 1`] = `
<div>
  <header
    class="header"
  >
    <h1
      class="header-title"
    >
      Math Magician App
    </h1>
    <nav>
      <ul>
        <li>
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/calculator"
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            href="/quotes"
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </header>
</div>
`;
